README.md

```diff
@@ -1,8 +1,32 @@
 # SymPy Paper
 
-[![Join the chat at https://gitter.im/asmeurer/sympy-paper](https://badges.gitter.im/asmeurer/sympy-paper.svg)](https://gitter.im/asmeurer/sympy-paper?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
+This is the source repository for the paper:
+
+> SymPy Authors, "SymPy: Symbolic Computing in Python", 2016.
+
+This repository contains or links to all of the information needed to reproduce
+the results in the paper.
+
+The latest rendered version of the PDF can be viewed via the
+[ShareLaTeX](http://sharelatex.com) CI system:
 
 [![PDF Status](https://www.sharelatex.com/github/repos/asmeurer/sympy-paper/builds/latest/badge.svg)](https://www.sharelatex.com/github/repos/asmeurer/sympy-paper/builds/latest/output.pdf)
 
+# Contributing
+
+Join our chat room to discuss contributions:
+
+[![Join the chat at https://gitter.im/asmeurer/sympy-paper](https://badges.gitter.im/asmeurer/sympy-paper.svg)](https://gitter.im/asmeurer/sympy-paper?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
+
+# License
+
+The contents of this repository is licensed under the [Creative Commons
+Attribution 4.0 International
+License](http://creativecommons.org/licenses/by/4.0).
+
+[![CCBY](https://i.creativecommons.org/l/by/4.0/80x15.png)](http://creativecommons.org/licenses/by/4.0)
```

Comment:
I couldn't find if CC-BY 4.0 is open source. The only mention I found is here:

https://wiki.debian.org/DFSGLicenses#Creative_Commons_Attribution_Share-Alike_.28CC-BY-SA.29_v4.0

which says that CC-BY-SA 4.0 is compatible with DFSG, which is essentially the same as the open source definition at opensource.org. However, I found that a previous version of the license, mainly CC-BY 1.0 is not open source:

https://wiki.debian.org/DFSGLicenses#Creative_Commons_Attribution_License_.28CC-by.29.2C_v1.0

So for these reasons, why cannot we just use an MIT or BSD license?